### DISCUSSION TITLE
Is Root Access to the Outdoor Base Station possible? (WVODB1) with camera model (WVOD1B1)

### DISCUSSION THREAD

**Discussion by markthomas93** [gtxaspec/wz_mini_hacks#694] [Q&A]
I've got a spare base station (WVODB1) that was working great, but for some reason is now stuck in the "blue light blinking" loop.
After checking Github & Reddit, it looks like the Wyze App may have sent an update to the Outdoor Cam Router with faulty firmware.

**Question**: _Would using the firmware flashing process here on this repo give access to the router?_

The Outdoor Cam Router does accept and show the correct lights when being flashed with official router firmware.
But I've been unsuccessful with getting back to "default" - and bought another Outdoor Cam router to use the official way.

Looking at trying to use this base station as an offline router for more Wyze Outdoor Cam (V1) (WVOD1B1) cameras to experiment with.

Thought I would ask first and see if someone else has already been down this path.

**Comment by virmaior** [comment]
doubt it.

maybe i'm not reading well (https://fccid.io/2ANJHWVODB1) but that doesn't seem to have an ingenic T20 or T31
